dnf-behave-tests/dnf/dnf-automatic/error-report.feature

```diff
@@ -0,0 +1,25 @@
+@no_installroot
+Feature: dnf-automatic reports an error when transaction failed
+
+
+Background:
+Given I delete file "/etc/yum.repos.d/*.repo" with globs
+  And I create file "/etc/dnf/dnf.conf" with
+    """
+    [main]
+    plugins=0
+    """
+
+
+@bz2170093
+Scenario: dnf-automatic reports an error when package installation failed
+  Given I use repository "dnf-ci-automatic-update"
+    And I successfully execute dnf with args "install test-1.0"
+   When I execute dnf-automatic with args "--installupdates"
```

Comment:
From test description it is not clear why the command is supposed to fail. What about to add a comment that the step will try  to install test-1-1 with broken scriptlet. The expected result is clear from test but not why it is expected.